cmd/fireeye2nvd/fireeye2nvd.go

```diff
@@ -55,18 +56,21 @@ func main() {
 	sinceUnix := flag.Int64("since_unix", 0, "Unix timestamp since when should we download. If not set, downloads all available data")
 	sinceDuration := flag.String("since", "", "Golang duration string, overrides -since_unix flag")
 	dontConvert := flag.Bool("dont_convert", false, "Should the feed be converted to NVD format or not")
+	stats.Global.AddFlags()
 	flag.Usage = func() {
 		fmt.Fprintf(os.Stderr, "Usage: %s [flags]\n", os.Args[0])
 		flag.PrintDefaults()
 		os.Exit(1)
 	}
 	flag.Parse()
+	defer stats.Global.WriteAndLogError()
 
 	since := *sinceUnix
 	if *sinceDuration != "" {
 		dur, err := time.ParseDuration("-" + *sinceDuration)
 		if err != nil {
-			log.Fatalln(err)
+			log.Println(err)
```

Comment:
We loose stacktrace with this change... What about that idea of yours to defer the `recover()` call?